config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
 Style/Documentation:
   Enabled: false
 
+Style/EmptyLinesAroundBlockBody:
+  Enabled: false
+
 Style/EmptyLinesAroundClassBody:
   Enabled: false
 
 Style/EmptyLinesAroundModuleBody:
   Enabled: false
 
+Style/ExtraSpacing:
+  Enabled: false
+
+Style/IndentArray:
```

Comment:
I would prefer to leave IndentArray set to false, and use PR to ensure code is readable.  If it is enabled, the flavors avail are:  

```ruby
# Checks the indentation of the first element in an array literal.
Style/IndentArray:
  # The value `special_inside_parentheses` means that array literals with
  # brackets that have their opening bracket on the same line as a surrounding
  # opening round parenthesis, shall have their first element indented relative
  # to the first position inside the parenthesis.
  #
  # The value `consistent` means that the indentation of the first element shall
  # always be relative to the first position of the line where the opening
  # bracket is.
  #
  # The value `align_brackets` means that the indentation of the first element
  # shall always be relative to the position of the opening bracket.
  EnforcedStyle: special_inside_parentheses
  SupportedStyles:
    - special_inside_parentheses
    - consistent
    - align_brackets
  # By default, the indentation width from Style/IndentationWidth is used
  # But it can be overridden by setting this parameter
  IndentationWidth: ~
```

I'd be more comfy with this rule if we upped the IndentationWidth for Array --  I think the code is more readable when the array elements are indented further than the following code.